src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -223,6 +230,132 @@ void refuseAttemptToMoveRequestBeingFulfilledFromOneOfTheTopPositionsWhenTlrEnab
+    final IndividualResource proxy = usersFixture.undergradHenry(user -> user.withPatronGroupId(staffGroupId));
+    proxyRelationshipsFixture.nonExpiringProxyFor(sponsor, proxy);
+
+    checkOutFixture.checkOutByBarcode(smallAngryPlanet);
+
+    requestsFixture.place(
+      new RequestBuilder()
+        .recall()
+        .itemRequestLevel()
+        .withRequestDate(ZonedDateTime.of(2017, 7, 22, 10, 22, 54, 0, UTC))
+        .forItem(smallAngryPlanet)
+        .withInstanceId(smallAngryPlanet.getInstanceId())
+        .by(sponsor)
+        .proxiedBy(proxy)
+        .fulfilToHoldShelf()
+        .withRequestExpiration(LocalDate.of(2017, 7, 30))
+        .withHoldShelfExpiration(LocalDate.of(2017, 8, 31))
+        .withPickupServicePointId(servicePointsFixture.cd1().getId())
+        .withTags(new RequestBuilder.Tags(asList("new", "important")))
+        .withPatronComments("I need the book"));
+
+    JsonArray requests = requestQueueFixture.retrieveQueueForItem(smallAngryPlanet.getId().toString()).getJsonArray("requests");
+
+    JsonObject request = requests.getJsonObject(0);
+    assertThat(request.containsKey("instance"), is(true));
+    JsonObject instance = request.getJsonObject("instance");
+    assertThat(instance.containsKey("title"), is(true));
+    assertThat(instance.containsKey("identifiers"), is(true));
+    assertThat(instance.containsKey("contributorNames"), is(true));
+    assertThat(instance.containsKey("publication"), is(true));
+    assertThat(instance.containsKey("editions"), is(true));
+
+    assertThat(request.containsKey("item"), is(true));
+    JsonObject item = request.getJsonObject("item");
+    assertThat(item.containsKey("barcode"), is(true));
+    assertThat(item.containsKey("location"), is(true));
+    assertThat(item.containsKey("enumeration"), is(true));
+    assertThat(item.containsKey("volume"), is(true));
+    assertThat(item.containsKey("chronology"), is(true));
+    assertThat(item.containsKey("status"), is(true));
+    assertThat(item.containsKey("callNumber"), is(true));
+    assertThat(item.containsKey("copyNumber"), is(true));
+
+    assertThat(request.containsKey("loan"), is(true));
+    JsonObject loan = request.getJsonObject("loan");
+    assertThat(loan.containsKey("dueDate"), is(true));
+
+    assertThat(request.containsKey("requester"), Is.is(true));
+    JsonObject requester = request.getJsonObject("requester");
+    assertThat(requester.containsKey("lastName"), is(true));
+    assertThat(requester.containsKey("firstName"), is(true));
+    assertThat(requester.containsKey("middleName"), is(false));
+    assertThat(requester.containsKey("barcode"), is(true));
+    assertThat(requester.containsKey("patronGroup"), is(true));
+    assertThat(requester.containsKey("patronGroup"), Is.is(true));
+
+    assertThat(request.containsKey("proxy"), Is.is(true));
+    final JsonObject proxySummary = request.getJsonObject("proxy");
+    assertThat(proxySummary.containsKey("patronGroup"), Is.is(true));
```

Comment:
why do you use the different matchers?

Reply:
Is() and UUIDMatcher.is() asserts differently, if I use is(), then returns with <> symbols